fmriprep/workflows/fieldmap/phdiff.py

```diff
@@ -131,6 +132,11 @@ def _pick1st(inlist):
 # Helper functions
 # ------------------------------------------------------
 
+@due.dcite(
+    Doi('10.1006/nimg.2001.1054'),
+    description="Conversion of the input phase-difference map into a fieldmap in Hz",
+    tags=['edu']
+)
```

Comment:
Sure:

```Python
from nipype.pipeline import engine as pe
from nipype.interfaces import utility as niu

@due.dcite(...)
def identfunc(x):
    return x

fnode = pe.Node(niu.Function(function=identfunc), name='fnode')
fnode.inputs.x = 1
fnode.run()
```

The specific problem is that nipype serializes the function as a string, and then reconstitutes it when it's time to run, which doesn't play nicely with decorators.